src/main/resources/templates/lfp/home.html

```diff
@@ -7,128 +7,76 @@
 <head>
     <title>Home</title>
 </head>
-<div id="temp-home-main-content"  layout:fragment="content">
+<div id="content" layout:fragment="content">
 
     <form th:action="@{''}" method="post" class="form" id="home">
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
-                <h1 class="govuk-heading-xl">
+                <h1 class="govuk-heading-xl" id="main-heading">
 
                     Pay a late filing penalty
 
                 </h1>
                 <p class="govuk-body-l">Use this service to pay a company's late filing penalty.</p>
-                <p class="govuk-body">You cannot use this service to pay:</p>
-                <ul class="govuk-list govuk-list--bullet">
-                    <li>a penalty that has been passed to a debt recovery agent</li>
-                    <li>a penalty under an instalment plan</li>
-                    <li>the remaining balance of a part-paid penalty</li>
-                </ul>
+
                 <p class="govuk-body">You'll need:</p>
-                <ul class="govuk-list govuk-list--bullet">
+                <ul class="govuk-list govuk-list--bullet" id="list-requirements">
                     <li>the company number</li>
                     <li>the reference number on the penalty notice</li>
-                    <li>an email address or Companies House account</li>
+                    <li>a Companies House account - you'll be able to create one if you do not have one already</li>
                     <li>a credit or debit card</li>
                 </ul>
-                <p class="govuk-body">It takes around 5 minutes to pay.</p>
-
 
+                <p class="govuk-body">You can <a class="govuk-link govuk-link--no-visited-state" id="appeals-link" href="/appeal-a-penalty/">appeal a late filing penalty</a>
+                    , but you'll need to show that the reasons for filing late were exceptional.
+                </p>
 
                 <div class="form-group">
                     <input id="next-button" class="govuk-button" data-event-id="lfp - Start now" onclick="_paq.push(['trackGoal', 3]);" type="submit" role="button" value="Start now"/>
```

Comment:
Is it ok for us to have JS events in the html file?

Reply:
According to Tori:
"just had a look now and I think most of CHS services use onclick for the Piwik goal inside the HTML so that should be fine".

Reply:
Okay great, as we're moving away from piwik to matomo, will this be okay to be removed?

Reply:
I would have thought that Piwiki to Matomo is a switch of where the events go, more than anything else, so this code would remain...
Double-checked with Tori & yup: keep the code. Her answer:

> All of the setup for Piwik will be exactly the same as Matomo, only thing that will change is Helen/Rachel will provide you with a different goal ID if they generate a new one in Matomo UI, and also adding config into your app_env to point to Matomo with a specific site ID.
> 
> Just a side note, the reason why we’re adding the config into our app_env instead of changing the global_env is because the secure project (Apply to be secure) still needs to use Piwik as they’re not sure if Matomo is secure enough as it is hosted on AWS and also because Helen & Rachel wants to move services over to Matomo one by one.

Reply:
Great, will make sure it stays!